lib/relation-definition.js

```diff
@@ -2458,20 +2458,6 @@ EmbedsMany.prototype.prepareEmbeddedInstance = function(inst) {
         process.nextTick(callback);
       }
     };
-    var originalTrigger = inst.trigger;
-    inst.trigger = function(actionName, work, data, callback) {
-      if (typeof work === 'function') {
-        var originalWork = work;
-        work = function(next) {
-          originalWork.call(this, function(done) {
-            inst.triggerParent(actionName, function(err, inst) {
```

Comment:
This change was added so that when saving an embedded model, it also correctly updates the parent model.

https://github.com/strongloop/loopback-datasource-juggler/commit/96a276a12b4a75a85b1092ccbd4ad0b31c5fd056